### ISSUE TITLE
No Longer Supported + Complete lack of Guide???

### ISSUE THREAD

**Issue by n3rdcub** [TransposonY/GestureSign#118]
First of all rather unique app. strange these are rare and hard to find. however, no updates since over a year ago is this still supported?

Also what is the deal with absolute lack of any documentation? No way to tell what's double press, single, multiple fingers, etc. For example how in the world is somebody supposed to automatically know ITS TWO FINGERS for the browser launch? 
when you receive a new product do you immediately throw out the instructions?

**Comment by jclavier44** [comment]
And you, when you get a free beer you spit it to the one who offered it to you?
The software is so easy to configure, it does not even need a doc, unless for people who do not take time to search in it.
If you need instructions, feel free to fork and improve it man!

**Comment by Etrit1** [comment]
It still works on Win11 with a touchpad and on touch screens.  Therefor, there is no need to update it.


As for documentation?   Come on, it's FREE, made by ONE person and it's easy enough to figure things out.  

I mean really, click on the gestures tab and you can figure out how many figures it takes. 

I will say that it would be nice if the dev at least updated the homepage and was like, "It still works on Win11 so we're all good."

**Comment by khall187** [comment]
Agree, this guy comes across as an ungrateful jerk. This app turned my laptop into a powerhouse of productivity and I'm thankful for it! My only 'wish' would be one last update to confirm it works with Windows 11 (it does), perhaps a dark mode toggle, but most importantly (to the OP's point) it'd be nice to have the Wiki/Help section updated, currently if you hit the "?" button (help), it goes to a broken link.
